js/js_notwasm.go

```diff
@@ -181,7 +181,7 @@ func ValueOf(x interface{}) Value {
 		return x.Value
 	case nil:
 		return Null()
-	case bool, int, int8, int16, int32, int64, uint, uint8, uint16, uint32, uint64, float32, float64, unsafe.Pointer, string:
+	case bool, int, int8, int16, int32, int64, uint, uint8, uint16, uint32, uint64, float32, float64, unsafe.Pointer, string, map[string]interface{}:
```

Comment:
and `[]interface{}`?

https://golang.org/pkg/syscall/js/?GOOS=js&GOARCH=wasm#ValueOf

Reply:
And, please add tests.